crates/nu-cli/src/completions/completion_common.rs

```diff
@@ -273,33 +274,32 @@ pub fn adjust_if_intermediate(
 pub fn sort_suggestions(
     prefix: &str,
     items: Vec<SemanticSuggestion>,
-    sort_by: SortBy,
+    options: &CompletionOptions,
 ) -> Vec<SemanticSuggestion> {
-    sort_completions(prefix, items, sort_by, |it| &it.suggestion.value)
+    sort_completions(prefix, items, options, |it| &it.suggestion.value)
 }
 
 /// # Arguments
 /// * `prefix` - What the user's typed, for sorting by Levenshtein distance
```

Comment:
Is Levenshtein still a thing?

Reply:
Good catch. Yeah, I just missed that comment, Levenshtein distance is gone now.